### ISSUE TITLE
1.19.3 Issues

### ISSUE THREAD

**Issue by Raycoms** [ldtteam/Domum-Ornamentum#120]
Creative Tab:

Creative tabs doors are still of
and fancy doors are missing in the creative tab
fancy trapdoors too
and panels too

And thin framed walls got rendering issues with the model

**Comment by Moobien** [comment]
Fancy door item doesn't match in world block

**Comment by Moobien** [comment]
need framed panels

**Comment by lamppost253** [comment]
The brick/stone brick variants added by DO are all listed as having incorrect recipes. This includes: brown, beige, cream, and sand bricks. All 8 of these recipes call for 4x of the base material (e.g. terracotta, sand, gravel, and sandstone) instead of the new planned recipes.

**Comment by Moobien** [comment]
> The brick/stone brick variants added by DO are all listed as having incorrect recipes. This includes: brown, beige, cream, and sand bricks. All 8 of these recipes call for 4x of the base material (e.g. terracotta, sand, gravel, and sandstone) instead of the new planned recipes.

What planned recipes?

**Comment by Talyda** [comment]
> What planned recipes?

See here Moobien 
https://github.com/ldtteam/Domum-Ornamentum/issues/126
They were asked to make an issue, then asked to comment, after they had made the issue, so did both
They mean the actual recipes are wrong

**Comment by DBotThePony** [comment]
Barrels are also missing from creative menu (and hence missing from JEI)

**Comment by Raycoms** [comment]
Issues with doors rendering offcenter and panels not rendering at all too.

**Comment by mikebozo** [comment]
Hello, the floating carpet recipe does not work in version domum_ornamentum-1.19.3-1.0.86-ALPHA-universal (the latest one)

**Comment by mikebozo** [comment]
Just to add to this, the standing and laying barrels dont appear in aforementioned version as craftable, or in JEI. I can spawn them in using the /give command. so they are in there, just dont show up. :) 

Edit : All forms of Panels are invisible in the architect cutter.

**Comment by Raycoms** [comment]
They are probably in there, just not in the creative tab.

**Comment by marchermans** [comment]
## TLDR:

### Rendering:
- [x] Any door renders weird in the inventory
- [x] Panels are not rendering at all in the inventory

### Creative Tabs:
- [x] Barrels are missing

### Broken recipes:
- [x] Floating carpet
